src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -113,7 +113,7 @@ public function request(string $method, string $url, array $options = []): Respo
         $url = implode('', $url);
 
         if (!isset($options['normalized_headers']['user-agent'])) {
-            $options['normalized_headers']['user-agent'][] = $options['headers'][] = 'User-Agent: Symfony HttpClient/Curl';
+            $options['headers'][] = 'User-Agent: Symfony HttpClient/Curl';
```

Comment:
is this change related to this gzip issue ?

Reply:
not directly